app/agents/validator.py

```
@@ -0,0 +1,272 @@
+"""
+Agente Validator - Valida datos específicos del usuario
+Integra con las funciones de validación existentes en utils/validators.py
+"""
+
+import re
+from typing import Any, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)